ShowMyCursedRealm/room.aadl

```diff
@@ -1,45 +1,94 @@
 package room
 public
+with threads;
+with deployment;
+	
 	
 	device camera
 		features
-			video: out data port;
-			camera_on: in event port;
+			video_stream: out data port threads::Video_Stream;
+			camera_on_off: in event port;
 	end camera;
 	
 	device movement_sensor
 		features
-			movement_detected: out event port;
+			movement_data: out data port threads::BooleanValue;
```

Comment:
Czy `event` nie jest odpowiednikiem `boolean'a`?
Bo trochę nie rozumiem zmiany.

Reply:
Trochę tak,

W zasadzie mogłoby zostać na tych wszystkich że nie przesyłają danych tylko `out event port`, tylko nie wiem jak to zrobić w kodzie

Reply:
wszystkich sensorach*